pypeit/core/gui/identify.py

```diff
@@ -611,6 +612,12 @@ def operations(self, key, axisID):
             self.replot()
         elif key == 'l':
             self.load_IDs()
+        elif key == 'm':
+            msgs.info("Selecting line")
```

Comment:
I'm not sure this info message is needed (the user is probably not looking at the command line).

I think you can remove the line `self._addsub = 1`

Is this so that you can select a line without having to use the mouse?

Reply:
removed.

and, yes,  the mouse doesn't work as well on my Linux platform.